lib/raven/integrations/sidekiq.rb

```diff
@@ -3,56 +3,57 @@
 
 module Raven
   class Sidekiq
-    def call(_worker, msg, _queue)
-      started_at = Time.now
-      yield
-    rescue Exception => ex
-      Raven.capture_exception(ex, :extra => { :sidekiq => msg },
-                                  :time_spent => Time.now - started_at)
-      raise
-    ensure
-      Context.clear!
-      BreadcrumbBuffer.clear!
+    ACTIVEJOB_RESERVED_PREFIX = "_aj_".freeze
+
+    def call(ex, context)
```

Comment:
this is a really simple middleware that can be tested in isolation from sidekiq, should add some tests